system_test/test_compatibility_checker_server.py

```diff
@@ -63,17 +60,18 @@ def tearDown(self):
         # Kill the flask application process
         os.killpg(os.getpgid(self.process.pid), signal.SIGTERM)
 
-    @retry(wait_fixed=RETRY_WAIT_PERIOD,
-           stop_max_attempt_number=RETRY_MAX_ATTEMPT)
+    @retry(
+        wait_fixed=RETRY_WAIT_PERIOD, stop_max_attempt_number=RETRY_MAX_ATTEMPT)
     def test_check_compatibility(self):
         response = requests.get('{}?package={}&python-version={}'.format(
             BASE_URL, PACKAGE_FOR_TEST, 3))
         status_code = response.status_code
         content = response.content.decode('utf-8')
 
-        content_dict = ast.literal_eval(content.replace(
-            'true', '"true"').replace(
-            'false', '"false"').replace('null', '"null"'))
+        content_dict = ast.literal_eval(
+            content.replace('true', '"true"').replace('false',
+                                                      '"false"').replace(
+                                                          'null', '"null"'))
```

Comment:
It could just be that this one line is trying to do too much and should actually be broken up, but it just looks really weird at the moment.